lib/db.js

```diff
@@ -63,6 +63,10 @@ function getMongoConnectionUrl(cb) {
     return mongoConnectionStringOS2(cb);
   } else if ("openshift3" === process.env.FH_MBAAS_TYPE) {
     return mongoConnectionStringOS3(cb);
+  } else if (process.env.FH_USE_LOCAL_DB) {
+    // Backwards compatible, hard coded localhost for local development
+    process.env.FH_MONGODB_CONN_URL = 'mongodb://localhost:27017';
```

Comment:
is it possible the mongodb url maybe different locally? Maybe check if there is another mongodb url env var first and then default to the local one? WDYT?